assets/js/main.js

```diff
@@ -392,6 +392,7 @@ class Carousel {
 }
 
 var myCarousel = [new Carousel($("#CarouselPhoto"))]
```

Comment:
Just append to the list.

Reply:
If you want to use two list, then you have to modify line249 too.

Reply:
Now appended to the list.